### ISSUE TITLE
wrong dependency used in plugin-eslint  

### ISSUE THREAD

**Issue by AngusFu** [rollup/plugins#645]
- Rollup Plugin Name:  @rollup/plugin-eslint

### Expected Behavior

It should work!

### Actual Behavior

> [!] Error: Cannot find module 'rollup-pluginutils'

### Additional Information

![image](https://user-images.githubusercontent.com/8180186/99027866-3e4f0980-25a9-11eb-9735-26a3852b8ae4.png)
![image](https://user-images.githubusercontent.com/8180186/99027884-4d35bc00-25a9-11eb-80bf-c168c0f1db87.png)

**Comment by UnleashSpirit** [comment]
Hi,
To up the issue, I have the same error

```json
"devDependencies": {
    "@rollup/plugin-commonjs": "^16.0.0",
    "@rollup/plugin-eslint": "^8.0.0",
    "@rollup/plugin-node-resolve": "^10.0.0",
    "@rollup/plugin-typescript": "^6.1.0",
    "@rollup/pluginutils": "^4.1.0",
    "rollup": "^2.33.3",
    "rollup-plugin-terser": "^7.0.2",
    "tslib": "^2.0.3",
    "typescript": "^4.1.2"
  }
```

Workaround for now
`npm install rollup-pluginutils --save-dev`

**Comment by shellscape** [comment]
Fixed in #644
